### DISCUSSION TITLE
Help identify protocol ar09nxfpewqneu

### DISCUSSION THREAD

**Discussion by lucapepe97** [omerfaruk-aran/esphome_samsung_hvac_bus#156] [General]
Hello, i have a single split samsung ac 9000 btu.
The code both the compressor and the split is ar09nxfpewqneu.
I cannot find in any documentation if the protocol is NASA or NON NASA.
Thanks for the help.

**Comment by jszkiela72** [comment]
Probably NASA.

![image](https://github.com/user-attachments/assets/b0e7bb5b-648d-4bcc-98b1-826c6fd068ae)

**Comment by tfoutfou** [comment]
R32 unit , NASA ,   99,5 % chance

i can pretty much say with confidence that non-NASA is so old it was droped before the unit became R32
